Library/Homebrew/test/test_dependency_collector.rb

```diff
@@ -50,29 +51,34 @@ def test_no_duplicate_requirements
   def test_requirement_tags
     @d.add :x11 => "2.5.1"
     @d.add :xcode => :build
-    assert_empty find_requirement(X11Requirement).tags
+    req = OS.mac? ? X11Requirement : XorgRequirement
+    assert_empty find_requirement(req).tags
     assert_predicate find_requirement(XcodeRequirement), :build?
   end
 
   def test_x11_no_tag
     @d.add :x11
-    assert_empty find_requirement(X11Requirement).tags
+    req = OS.mac? ? X11Requirement : XorgRequirement
+    assert_empty find_requirement(req).tags
   end
 
   def test_x11_min_version
+    skip "Only tracked on Mac OS" unless OS.mac?
```

Comment:
Perhaps changes this comment to `"XQuartz versions are relevant only on Mac OS"`.